### PR TITLE
add CSS changes

### PR DESCRIPTION
**Pull request by sanyamkamat**

### CODE REVIEW

File: app/app.component.ts [hunk 1]
```diff
@@ -2,16 +2,7 @@ import { Component } from '@angular/core';
...
-      <h1>My First Angular 2 App</h1>
-
-      <a routerLink="/products">Products</a>
-      <a routerLink="/login">Login</a>
-      <a routerLink="/register">Register</a>
-      <a routerLink="/cart">Cart</a>
-      <a routerLink="/slowcart">Slow Cart</a>
-      <router-outlet></router-outlet>
-    `
+    templateUrl: './app/header.html'
```
Comment (shyamseshadri) [inline app/app.component.ts]: app.html

File: app/products/product-list-item.html [hunk 2]
```diff
@@ -1,6 +1,9 @@
-<div class="product-item" (click)="onSelect(product)"
-     [class.soldout]="product.isSoldOut">
-  <h3>{{product.name}}</h3>
-  <div>{{product.price | currency:'INR':true:'1.2-2'}}</div>
+<div class="product-item clearfix col-xs-6 col-sm-4 cold-md-3 clearfix">
+  <div class="product-item-wrapper"
+       (click)="onSelect(product)"
+       [class.soldout]="product.isSoldOut">
+    <h3>{{product.name}}</h3>
+    <img src="{{product.imgUrl}}" alt="">
```
Comment (shyamseshadri) [inline app/products/product-list-item.html]: noooooo

[src]="product.imgUrl"

File: app/products/product-list.component.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ import { ProductService } from './product.service';
 })
 export class ProductListComponent implements OnInit {
 
+  @HostBinding('class') classes = 'row';
```
Comment (shyamseshadri) [inline app/products/product-list.component.ts]: wtf is this??

File: app/products/product-list-item.html (hunk 2 above)
Comment (sanyamkamat) [reply app/products/product-list-item.html]: I had checked angular.io for this. Just picked up from first example.
Will change it.

File: app/products/product-list.component.ts (hunk 3 above)
Comment (sanyamkamat) [reply app/products/product-list.component.ts]: A way to add classes to your components.
Suggest how to add wrapper for list of products with class ".row"

**Review by sanyamkamat — COMMENTED** [review]
Added review changes

**Pull request closed without merging**